WebARonARKit/WebARonARKit.js

```diff
@@ -798,30 +833,14 @@
     }
 
     WebARonARKitVRDisplay.anchors_ = data.anchors;
```

Comment:
Will anchors still be passed from the native side in each update call?

Reply:
Yes. Do you want me to change it to register event handlers and keep its own list in JS? I can do that.